func_sig_registry/utils/solidity.py

```diff
@@ -204,3 +208,7 @@ def normalize_function_signature(raw_signature: str) -> str:
         raise ValueError('function args contain non-standard types') from e
     else:
         return f'{fn_name}{args_tuple_type.to_type_str()}'
+
+#TODO
```

Comment:
I'm not strongly opposed to doing things this way: stubbing something out and filling it in later...

However, I think what you might do once you get these filled in (and add relevant tests) is to open a PR that contains only this normalization/validation logic without the database model and we can review that in isolation.  This branch can then just be rebased on top of that functionality, after which it can be reviewed and merged on its own.

I think that in this case your current approach would probably be fine, in the more general case, you could find something wrong with your database model in the testing of these validators/normalizers which wasn't obvious in your initial database representation.